### DISCUSSION TITLE
Merge statistics

### DISCUSSION THREAD

**Discussion by riksarchen** [coinfoundry/miningcore#956] [Q&A] [answered]
Hello,
I have mined on two different addresses. Is it possible to merge statistics from the old address into the new by altering the database?

**Comment by oliverw** [comment, marked as answer]
Should work.

**Reply by riksarchen** [reply]
Do you have any suggetions on command? I'm newbie when it comes to databases

**Reply by oliverw** [reply]
`UPDATE minerstats SET miner = '<new address>' WHERE miner = '<old address>';`

**Reply by riksarchen** [reply]
Getting this error 
ERROR:  type " " does not exist

**Reply by oliverw** [reply]
> Getting this error
> ERROR: type " " does not exist

Do not use double quotes.

**Reply by riksarchen** [reply]
This command
miningcore=# UPDATE minerstats SET miner = 'vtc1qcrn34kay4c7kegv349
pv0eknmdjkxudw8upyel' WHERE miner = 'VhrFrQxJm4BQgu4QZiGjxRSsrTq3Co
hnWk';

Gets the error

**Reply by oliverw** [reply]
> This command
> miningcore=# UPDATE minerstats SET miner = 'vtc1qcrn34kay4c7kegv349
> pv0eknmdjkxudw8upyel' WHERE miner = 'VhrFrQxJm4BQgu4QZiGjxRSsrTq3Co
> hnWk';
> 
> Gets the error

Hmm. Looks good to me.

**Reply by riksarchen** [reply]
Something wrong on my side. Tried it again and update 19k posts..

**Reply by riksarchen** [reply]
Hm, stats did not update though. Old stats didnt show under api

**Comment by riksarchen** [comment]
I ran this command aswell and now I got all statistics with me

miningcore=# UPDATE payments SET address = 'vtc1qcrn34kay4c7kegv349
pv0eknmdjkxudw8upyel' WHERE address = 'VhrFrQxJm4BQgu4QZiGjxRSsrTq3
CohnWk';
UPDATE 29
miningcore=#

**Comment by riksarchen** [comment]
Thanks a lot for your help! Much appreciated

**Reply by oliverw** [reply]
You're welcome 😄

**Comment by riksarchen** [comment]
For future referenser also updated balancechanges table and blocks tables (finder column)

This can be closed. Thanks for your Support
